.github/workflows/gosec.yml

```diff
@@ -0,0 +1,31 @@
+name: "Code Scanning - Gosec"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
```

Comment:
There should be a `go build` somewhere I believe, to take care of dependencies. Possibly `go get download` could work as well.